airflow/utils/task_group.py

```diff
@@ -453,6 +454,21 @@ def topological_sort(self, _include_subdag_tasks: bool = False):
 
         return graph_sorted
 
+    def get_task_dict(self) -> dict[str, AbstractOperator]:
+        """Returns a flat dictionary of task_id: AbstractOperator"""
+        task_map = {}
+
+        def build_map(dag_node):
+            if not isinstance(dag_node, TaskGroup):
```

Comment:
You want to flip this check, checking something is a task (AbstractOperator) instead of not a task group

Reply:
Done